internal/cmd/commands/database/init.go

```diff
@@ -176,6 +184,15 @@ func (c *InitCommand) Run(args []string) (retCode int) {
 		}()
 	}
 
+	if migrations.DevMigration != c.flagAllowDevMigration {
+		if migrations.DevMigration {
+			c.UI.Error("This version of the binary has unsafe dev database schema updates.  To proceed anyways please use the 'allow-development-migration' flag.")
```

Comment:
```suggestion
			c.UI.Error("This version of the binary has dev database schema updates which may not be supported in the next official release. To proceed anyways, please use the 'allow-development-migrations' flag.")
```
Note one of the changes in the text -- `allow-development-migrations` -- I think making it plural is probably correct since we always talk about them in terms of migrations, which covers both the single and plural cases.